src/views/tables/MarketsCompactTable.tsx

```diff
@@ -146,8 +146,8 @@ export const MarketsCompactTable = ({
   );
 
   const sortedMarkets = useMemo(() => {
-    if (sorting === MarketSorting.HIGHEST_CLOB_PAIR_ID) {
-      return filteredMarkets.sort((a, b) => b.clobPairId - a.clobPairId);
+    if (sorting === MarketSorting.RECENTLY_LISTED) {
+      return filteredMarkets.sort((a, b) => (a.line?.length ?? 0) - (b.line?.length ?? 0));
```

Comment:
huuuuuh what is line and why does it correlate with listing time?

Reply:
line is the 24H sparkline data. We are sorting by how many line items, so less than 24 means that it has been live for less than a day.